app/assets/javascripts/chf_image_viewer.js

```diff
@@ -124,10 +117,11 @@ ChfImageViewer.prototype.selectThumb = function(thumbElement) {
   var shouldShowInfo = thumbElement.getAttribute('data-member-should-show-info') == "true";
   var title = thumbElement.getAttribute('data-title');
   var linkUrl   = thumbElement.getAttribute('data-member-show-url');
+  var titleSource = thumbElement.getAttribute('data-title-source');
 
   $('.viewer-image').addClass('viewer-image-loading');
 
-  this.viewer.open(this.id2TileSource(id));
+  this.viewer.open(titleSource);
 
```

Comment:
i think my wires were crossed a bit. 'tile source' makes so much more sense; I kept wondering why it was called 'title source'.